gfx/display_servers/dispserv_win32.c

```diff
@@ -102,11 +102,11 @@ static void* win32_display_server_init(void)
 
 static void win32_display_server_destroy(void *data)
 {
-   dispserv_win32_t *dispserv = (dispserv_win32_t*)data;
-
-   if (win32_orig_width > 0 && win32_orig_height > 0)
+   if (win32_orig_width > 0 && win32_orig_height > 0 )
       video_display_server_switch_resolution(win32_orig_width, win32_orig_height,
-         0, 60);
+         60, 60);
+
+   dispserv_win32_t *dispserv = (dispserv_win32_t*)data;
```

Comment:
We compile C code as C89 for MSVC versions that are old. Variables need to be declared either at the top of a function or at the top of a block. By moving it like this, it would break all older MSVC versions right now.